website/CONTRIBUTE.md

```diff
@@ -0,0 +1,215 @@
+<!--
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+-->
+
+# Contribution Guide
+
+This guide consists of:
+
+- [Project structure](#project-structure)
+- [How to add a new blogpost](#how-to-add-a-new-blogpost)
+- [How to add a new page](#how-to-add-a-new-page)
+- [How to write in Hugo way](#how-to-write-in-hugo-way)
+  - [Define TableOfContents](#define-tableofcontents)
+  - [Language switching](#language-switching)
+  - [Code highlighting](#code-highlighting)
+  - [Adding class to markdown text](#paragraph)
+  - [Table](#table)
+  - [Github sample](#github-sample)
+  - [Others](#others)
+
+## Project structure
+
+```
+www/
+├── dist                                  # bundle files
+├── site
+│   ├── archetypes                        # frontmatter template
+│   ├── assets
+│   │   └── scss                          # styles
+│   ├── content                           # pages
+│   │   └── en
+│   │       ├── blog
+│   │       ├── community
+│   │       ├── contribute
+│   │       ├── documentation
+│   │       ├── get-started
+│   │       ├── privacy_policy
+│   │       ├── roadmap
+│   │       └── security
+│   │       └── _index.md
+│   ├── data
+│   ├── layouts                           # content template
+│   ├── static
+│   │   ├── downloads                     # downloaded files
+│   │   └── fonts
+│   │   └── images
+│   │   └── js
+│   └── themes
+│       └── docsy
+├── check-links.sh                        # links checker
+└── package.json
+```
+
+## How to add a new blogpost
+
+To add a new blogpost with pre-filled frontmatter, in `/www/site` run:
+
+```
+$ hugo new blog/my-new-blogpost.md
+```
+
+That will create a markdown file `/www/site/content/<LANGUAGE_VERSION>/blog/my-new-blogpost.md` with following content:
+
+```
+---
+title: "My New Blogpost"
+date: "2020-04-20T14:02:57+02:00"
+categories: 
+  - blog
+authors: 
+  - "Your Name"
+---
+```
+
+Below frontmatter, put your blogpost content. The filename will also serve as URL for your blogpost as `/blog/{filename}`.
```

Comment:
Please mention also `<!--more-->`. That's the delimiter between Summary and main content